apps/web/src/hooks/latte/index.ts

```diff
@@ -0,0 +1,9 @@
+'use client'
+
+export { useLatteChangeActions } from './useLatteChangeActions'
+export { useLatteChatActions } from './useLatteChatActions'
+export { useLatteDebugMode } from './useLatteDebugMode'
+export { useLatteProjectChanges } from './useLatteProjectChanges'
+export { useLatteThreadUpdates } from './useLatteThreadUpdates'
+export { useLoadThreadFromProviderLogs } from './useLoadThreadFromProviderLogs'
+export { useSyncLatteUrlState } from './useSyncLatteUrlState'
```

Comment:
Why do we need barrel files? I find it unnecessary.

Reply:
we don't really, was just easier for me, can remove

Reply:
🔥